### ISSUE TITLE
get rid of LD_LIBRARY_PATH requirement

### ISSUE THREAD

**Issue by cchriste** [SCIInstitute/ShapeWorks#125]
Adding link paths to the executables is the first step, and updating those link paths when bundling them for the installation is the next (I thought this was already an open issue since it's so important for user deployment).

**Comment by cchriste** [comment]
For most of our executables in install, this is already working. For the GUI executables, we should be packaging them as app bundles (in osx), for which there is clear documentation here: 
https://doc.qt.io/archives/qt-5.11/osx-deployment.html
https://medium.com/@donblas/fun-with-rpath-otool-and-install-name-tool-e3e41ae86172

For Linux and Windows, it's probably similar.

**Comment by akenmorris** [comment]
I'm not sure it makes sense to package ShapeWorksView2 as an app bundle.  At the moment it has to be run from the command line.

Also, I find that executables such as stl2ply which are linked to VTK (that we built with Qt) are requiring Qt libraries to run.

**Comment by cchriste** [comment]
Just to demonstrate this. From the install directory on OSX, this will fix ShapeWorksView2 so it runs from install. I'm out of time to make it more robust right now, but it's certainly possible.
```
cp -r /Users/cam/tools/Qt5.13.0/5.13.0/clang_64/lib/QtCore.framework .
cp -r /Users/cam/tools/Qt5.13.0/5.13.0/clang_64/lib/QtGui.framework ./lib
cp -r /Users/cam/tools/Qt5.13.0/5.13.0/clang_64/lib/QtWidgets.framework ./lib
cp -r /Users/cam/tools/Qt5.13.0/5.13.0/clang_64/lib/QtSql.framework ./lib
install_name_tool -add_rpath @executable_path/lib ShapeWorksView2
```

**Comment by akenmorris** [comment]
Right, that's why I brought up the possibility of writing our own installer script.

**Comment by cchriste** [comment]
Yes. It could be part of the CMakeLists.txt or a bash script (or batch file) called by cmake.

**Comment by akenmorris** [comment]
I've been trying to avoid that and having CMake take care of things for us.  Like I said, I haven't found a way.

I'm again questioning why the superbuild does "make install"  Seems like this is only something that 1-2 people would do in order to make the binaries for download.  Why would a SCI grad student want this huge duplicate of all binaries and libraries?

**Comment by cchriste** [comment]
There should be a working `make install` whether or not it's used by a grad student.

**Comment by akenmorris** [comment]
Working that can build downloadable installers from anyone's random setup?  Why?
